apps/slash-stories/src/Layout/HeaderTitle/HeaderTitle.stories.tsx

```diff
@@ -69,3 +89,42 @@ export const ComplexTitle: StoryObj<typeof HeaderTitle> = {
     ),
   },
 };
+
+export const ComplexTitleWithContentAndChildren: StoryObj<typeof HeaderTitle> =
+  {
+    args: {
+      contentLeft: (
+        <div className="af-title-bar__actions">
+          {leftActions.map(
+            ({ icon, id, title }: ComponentProps<typeof Action>) => (
+              <a
```

Comment:
as it is just a story i dont think it is useful but:

a list of link should be with following structure: <nav><ul/ol><li>